platforms/common/src/CommonKernels.cpp

```diff
@@ -4390,17 +4430,19 @@ CommonCalcCustomManyParticleForceKernel::~CommonCalcCustomManyParticleForceKerne
 
 void CommonCalcCustomManyParticleForceKernel::initialize(const System& system, const CustomManyParticleForce& force) {
     ContextSelector selector(cc);
+    const int tileSize = cc.getTileSize();
```

Comment:
This is another class that's computed with completely different code from standard nonbonded interactions.  It doesn't need to use the same tile size they do.